### PR TITLE
added a ground, needs get the texture working

### PR DESCRIPTION
**Pull request by MUYANGGUO**
test adding a level, needs working on texture, shadow not working. 
ground

### CODE REVIEW

**Comment by MUYANGGUO** [comment]
https://github.com/mrdoob/three.js/tree/dev/examples/jsm

Need to use this library. @lele94218

File: src/components/animation.tsx [hunk 1]
```diff
@@ -32,4 +87,4 @@ class Animation extends Component {
   }
 }
 
-export default Animation;
+export default Animation
```
Comment (trxue) [inline src/components/animation.tsx:90]: Why remove ;

Comment (MUYANGGUO) [reply src/components/animation.tsx:90]: typo

Comment (MUYANGGUO) [reply src/components/animation.tsx:90]: https://github.com/mrdoob/three.js/blob/dev/examples/jsm/controls/OrbitControls.js
we can use this kind of controller.

Comment (trxue) [reply src/components/animation.tsx:90]: Sounds good. Will take a look tomorrow

Comment (MUYANGGUO) [reply src/components/animation.tsx:90]: LMK

**Pull request merged**